src/notation/view/abstractnotationpaintview.cpp

```diff
@@ -94,6 +89,15 @@ void AbstractNotationPaintView::load()
 {
     TRACEFUNC;
 
+    //! NOTE For diagnostic tools
+    static bool once = false;
+    if (!once) {
+        once = true;
+        dispatcher()->reg(this, "diagnostic-notationview-redraw", [this]() {
+            scheduleRedraw();
+        });
+    }
+
```

Comment:
I'm afraid this will essentially break that tool: when you create a new score, there is already an AbstractNotationPaintView instance in the templates window, and that will register itself, and then the real notation paint view will fail to register itself. 
Perhaps you could create a new interface, that will register itself for the action and emit a notification that all AbstractNotationPaintView instances will listen to. However, that sounds a bit like overkill, for a simple dev-tool that we don't use often.

Reply:
fixed

Reply:
It looks like you didn't push this fix yet

Reply:
@igorkorsukov What about this? It looks like you didn't push the fixes that you made after my review

Reply:
hmm, mystics, it it should have been pushed, sorry (, I will push this fix in next PR

Reply:
https://github.com/musescore/MuseScore/pull/22955/files#diff-1ad85d6753e0a807acd0537fa45ee920e557f46ba99c86331e045fd37bec36abR93